src/_guides/language/evolution.md

```diff
@@ -270,6 +270,22 @@ To learn more about named parameters and arguments, see the
 
 ### Dart 2.18
 
+Dart 2.18 enhanced type inference by allowing information flow between arguments in generic function calls.
+Leaving argument type unspecified within some invocations previously triggered type errors,
+citing potential null occurrences.
+The compiler can now infer argument type from other values in an invocation,
+so specifying argument type inline is no longer necessary. 
+For more information,
+see the [Type system][] page.
+
+[Type system]: /guides/language/type-system
```

Comment:
@mit-mit would the [Type argument inference](https://dart.dev/guides/language/type-system#type-argument-inference) section specifically be a better place to link to for more information on this topic?

If so, could you please confirm none of the description there needs to be updated due to this enhancement?

Reply:
Yes, that would be a fine place for it